monkey/monkey_island/cc/ui/.babelrc

```diff
@@ -1,10 +1,15 @@
 {
   "presets": [
-    "es2015",
-    "stage-0",
-    "react"
+    [
+      "@babel/preset-env",
+      {
+        "useBuiltIns": "entry"
+      }
+    ],
```

Comment:
Can you tell me more about this change? preset-env by default parses into es2015 and what did stage-0 do ?

Reply:
stage-0 and es2015 were what babelrc transpiled into. All modern browsers already support it so there's nothing to transpire.
babel/preset-env lets babel target using browserslist (see change in package.json), which I defined to all browsers in use (0.25% and up). So not es2015.

Reply:
Debug output shows that currently this means the following results

Using targets:
{
  "chrome": "49",
  "edge": "18",
  "firefox": "73",
  "ie": "11",
  "ios": "12",
  "opera": "66",
  "safari": "12.1",
  "samsung": "10.1"
}